STMicroelectronics/STM32L4_L4+/app/az_ulib_dm/inc/_az_nx_blob_client.h

```diff
@@ -0,0 +1,42 @@
+#ifndef AZ_ULIB_DM_BLOB_CLIENT_H
+#define AZ_ULIB_DM_BLOB_CLIENT_H
+
+#include "az_ulib_result.h"
+#include "azure/az_core.h"
+#include "nx_api.h"
+#include "nx_web_http_client.h"
+#include "nx_wifi.h"
+#include "stm_networking.h"
+#include "wifi.h"
+
+
+#ifndef __cplusplus
+#include <stdint.h>
+#else
+#include <cstdint>
+#endif /* __cplusplus */
```

Comment:
You are using ThreadX standard for CHAR and ULONG. You can remove this include or use `stdint` standard instead.